requirements.txt

```diff
@@ -1,2 +1,24 @@
--r requirements-base.txt
-msgpack>=1.0.0
+#
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
+#
+#    pip-compile setup.py
+#
+certifi==2021.5.30
+    # via requests
+charset-normalizer==2.0.4
+    # via requests
+idna==3.2
+    # via requests
+msgpack==1.0.2
+    # via scrapinghub (setup.py)
+requests==2.26.0
+    # via scrapinghub (setup.py)
+retrying==1.3.3
+    # via scrapinghub (setup.py)
+six==1.16.0
+    # via
+    #   retrying
+    #   scrapinghub (setup.py)
+urllib3==1.26.6
+    # via requests
```

Comment:
What about removing this file entirely? As a library, we should keep dependencies in `setup.py` and let them be as flexible as possible.